corehq/apps/style/static/style/js/components/inline_edit_v2.js

```diff
@@ -98,8 +98,20 @@ hqDefine('style/js/components/inline_edit_v2.js', function() {
                     dataType: 'JSON',
                     data: data,
                     success: function (data) {
+                        function sanitizeValue() {
+                            var sanitize_properties = [".variable-app_name", ".variable-module_name", ".variable-form_name"];
+                            for(var i=0; i<sanitize_properties.length; i++) {
+                                if(data.update.hasOwnProperty(sanitize_properties[i])) {
+                                    var sanitized_value = $("<div/>").html(DOMPurify.sanitize(self.value())).text();
+                                    self.value(sanitized_value);
+                                    self.readOnlyValue = data.update[sanitize_properties[i]];
+                                    break;
+                                }
+                            }
+                        }
                         self.isSaving(false);
                         self.hasError(false);
+                        sanitizeValue();
```

Comment:
@orangejenny You are right. I sanitised this on the view first and then on the server so didn't realise i could just pick up the server value now. Updated!